### DISCUSSION TITLE
mount nfs time out

### DISCUSSION THREAD

**Discussion by mxolisimasango** [chpc-tech-eval/scc#156] [Q&A]
I’m attempting to mount an NFS share with the following command:

__________________________________________________________________________

However, my mount command is timing out. What could be causing this issue?

**Comment by WilliamMohlahlane** [comment]
### **ThunderBytes**

Hi, we are also having the same problem. Can your compute node connect to the chronny server?

**Comment by nyameko** [comment]
Please check your `nftables` configuration, make sure you've opened the correct ports.

This is an excellent point from @mohhlahlanewilliam , have you managed to get `chrony` working?

Also make sure you addresses are correct, in the `allowed section` in your exportsfs config file on your headnode, and make sure you've correctly configure your head node's ip in fstab, or the command line.

Make sure you're using the **INTERNAL** head node ip, not the public one.

**Comment by mbhonichau** [comment]
We've been stuck on the same problem also, 
![Screenshot 2024-07-11 211244](https://github.com/user-attachments/assets/a2f89d00-39f3-4b67-9836-bd48331de5eb)

**Reply by mbhonichau** [reply]
The compute node can not communicate with the headnode but the headnode can communicate with the compute node

![Screenshot 2024-07-11 232333](https://github.com/user-attachments/assets/ced50144-8a66-4aea-998c-004046b0098b)

**Reply by WilliamMohlahlane** [reply]
### **ThunderBytes**
What do you mean headnode can communicate with compute node. As in it can ping compute node?
Also please check if your 123 port is open or not. On your nftables make sure that the port is open so that chrony can recognise server and client. If nothing works, simply copy and paste the provided nftables template given to us in tutorial2. then retry some of the steps.
Get back to me
